src/pages/ListScholarships.jsx

```diff
@@ -92,6 +118,34 @@ function ListScholarships() {
         <Toolbar />
 
         <Container maxWidth="md" sx={{ flexGrow: 1 }}>
+          <Stack
```

Comment:
I kept it inside the container because once I moved it outside, the overflow prop would stop working. But I made is so that the tags wrap on md screens and higher